test/notification_test/notification_test.go

```diff
@@ -175,11 +175,11 @@ var _ = Describe("Notifications Suite", func() {
 				object := ctx.SMWithOAuth.GET("/v1/service_offerings").WithQuery("fieldQuery", "broker_id = "+id).
 					Expect()
 
-				so := object.Status(http.StatusOK).JSON().Object().Value("service_offerings").Array().First()
+				so := object.Status(http.StatusOK).JSON().Object().Value("items").Array().First()
 
 				servicePlanID := ctx.SMWithOAuth.GET("/v1/service_plans").WithQuery("fieldQuery", fmt.Sprintf("service_offering_id = %s", so.Object().Value("id").String().Raw())).
 					Expect().
-					Status(http.StatusOK).JSON().Object().Value("service_plans").Array().First().Object().Value("id").String().Raw()
+					Status(http.StatusOK).JSON().Object().Value("items").Array().First().Object().Value("id").String().Raw()
```

Comment:
This is a more genera comment - everywhere (in every test we do list) it will currently return a single page. However normally in those places we want all items. Right now all tests do work because the items that are created are leas than the default page size which is not reliable in general - so i think we will need a common utility used across all tests when talking about listing stuff that aggregates all pages

Reply:
This will be part of the token change since now we cannot iterate over the pages because the functionality is missing. With the PR that introduces the token and the real paging logic the tests must aggregate all the pages on each list call.